lib/SlackGhost.js

```diff
@@ -160,7 +162,7 @@ SlackGhost.prototype.updateAvatar = function(message, room) {
     });
 };
 
-SlackGhost.prototype.sendText = function(room_id, text) {
+SlackGhost.prototype.sendText = function(room_id, text, slackRoomID, slackEventTS) {
```

Comment:
Question, what's with the roomId and timestamps?

Reply:
My insane variable naming conventions?